source/MaterialXRuntime/RtToken.h

```diff
@@ -19,6 +19,10 @@ class RtToken;
 /// Token representing an empty string.
 extern const RtToken EMPTY_TOKEN;
 
+/// A list of tokens
+class RtToken;
+using RtTokenList = std::vector<RtToken>;
```

Comment:
Could move this to the bottom of this file where other token container types are defined.